backend/FwLite/LcmCrdt/CrdtMiniLcmApi.cs

```diff
@@ -317,7 +317,11 @@ public async Task BulkCreateEntries(IAsyncEnumerable<Entry> entries)
         var partsOfSpeech = await PartsOfSpeech.ToDictionaryAsync(p => p.Id, p => p);
         await dataModel.AddChanges(ClientId,
             entries.ToBlockingEnumerable()
-                .SelectMany(entry => CreateEntryChanges(entry, semanticDomains, partsOfSpeech))
+                .SelectMany(entry =>
+                {
+                    validators.ValidateAndThrow(entry).Wait(); //?
```

Comment:
We should just not use a blocking enumerable, and then we can use SelectManyAsyc (might not be the right name) and then we can await this properly

Reply:
Yeah, I can't figure out what the right way to handle this is 🙃

Reply:
yeah I can see why. There's a couple ugly bits. The main is that if you have an async method it can't just return an IAsyncEnumerable, it needs to use the `yield return` syntax, but you can't use that in a lambda method. So I just split the validation into it's own `SelectAwait` and used the sync version of `SelectMany` however that then wanted an async enum as a result instead of a normal `IEnumerable`